src/background/certificate/helpers/CertificateErrorAnalyzer.ts

```diff
@@ -2,10 +2,11 @@ import { ErrorFactory } from "../factories/ErrorFactory";
 
 export abstract class CertificateErrorAnalyzer {
   static analyzeError(requestDetails: {
+    url: string;
     frameId: number;
     error: string;
   }): Error | null {
-    if (requestDetails.frameId !== 0) {
+    if (requestDetails.frameId !== 0 || requestDetails.url === "about:blank") {

```

Comment:
Shouldn't this be a regex? There are many pages like about:blank? All in the about: and all in the chrome: namespaces.

Reply:
??